app/javascript/packs/application.tsx

```diff
@@ -483,6 +483,11 @@ import { PriceOption, PriceOptionProps } from '@/components/premium/PriceOption'
 
 document.addEventListener('turbo:load', () => {
   highlightAll()
+
+  if (window.location.pathname === '/dashboard') {
+    const siteHeader = document.querySelector('#site-header')
+    siteHeader?.classList.remove('theme-dark')
+  }
```

Comment:
I don't get what this does. What's special about the dashboard?

Reply:
I think some caching goes on, because when I logged in request.path in the ruby console, nothing appeared when visited dashboard, but worked fine for to other paths.